core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -161,15 +158,19 @@ class KiwixTextToSpeech internal constructor(
     )
     Log.d(TAG_KIWIX, "Audio Focus Requested")
     synchronized(focusLock) {
-      return@requestAudioFocus audioFocusRequest ==
-        AudioManager.AUDIOFOCUS_REQUEST_GRANTED
+      return@requestAudioFocus audioFocusRequest == AudioManager.AUDIOFOCUS_REQUEST_GRANTED
     }
   }
 
+  private fun abandonAudioFocus() {
```

Comment:
we can inline this method, only 1 usage and the method name just repeats the called method name

Reply:
@macgills is this okay?
```
private fun abandonAudioFocus() = am.abandonAudioFocus(onAudioFocusChangeListener)
```

Reply:
inline has a specific meaning. place your cursor on `abandonAudioFocus()`, press `ctrl+alt+n` and you will get inline options